module-2/backend/week_two.md

```diff
@@ -26,20 +34,50 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+Owner.where(team_id: 4)
```

Comment:
Owner wouldn't have `team_id`. The `belongs_to` exists on the table with the foreign key, so the `owner_id` would be on the `teams` table.

For question 3 would need something like the following:

```ruby
owner_id = Team.find(4).owner_id
Owner.find(owner_id)
```

With the additional code in 4 we can do the following:

```ruby
Team.find(4).owner
```